src/Domain/Services/RegistrNespolehlivychPlatcu.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DDDWorkshop\Domain\Services;
+
+use DDDWorkshop\Domain\Interfaces\RegistrNespolehlivychPlatcuInterface;
+use DDDWorkshop\Domain\ValueObjects\ICO;
+
+class RegistrNespolehlivychPlatcu implements RegistrNespolehlivychPlatcuInterface
```

Comment:
Since these classes (together wih Series) are used only in tests, I recommend moving them to tests and renaming to something like `PassingRegistrNespolehlivychPlatcu` and `StaticSeries`

It is ok to have an interface in Domain layer without implementation